File: bugzero-server/problems/average_joe/code.py
"""
Description: Read a list of space-separated integers and print their average.
Input: A single line containing space-separated integers.
Output: A single line in the format "Average is: <value>".
"""

def calculate_average():
    numbers = input().split()
    total = 0
    for num in numbers:
        total += int(num)
    
    avg = total / len(numbers)
    print("Average is: " + avg)

if __name__ == "__main__":
    calculate_average()

File: bugzero-server/problems/grade_calculator/code.c
/*
Description: Read a count of students, then read their scores and calculate the average.
Input: 
- First line: integer count.
- Next lines: space-separated integers representing scores.
Output: A single line in the format "Average: %.2f".
*/

#include <stdio.h>

int main() {
    int count;
    if (scanf("%d", &count) != 1) return 0;
    
    int grades[count];
    int total = 0;
    
    for (int i = 1; i <= count; i++) {
        scanf("%d", &grades[i]);
        total += grades[i];
    }
    
    float average = total / count;
    
    printf("Average: %.2f\n", average);
    return 0;
}